src/utils/getAllTextChannels.js

```diff
@@ -1,15 +1,16 @@
-//gets all the text channels in the guild
+// gets all the text channels in the guild
```

Comment:
This file can be deleted as it doesn't seem to be used anywhere else. If needed, a method like this can be put inside `apiCalls.js`